src/test/java/org/testeditor/web/xtext/index/persistence/GitServiceTest.xtend

```diff
@@ -3,58 +3,47 @@ package org.testeditor.web.xtext.index.persistence
 import com.google.inject.Guice
 import javax.inject.Inject
 import org.eclipse.jgit.api.Git
+import org.eclipse.jgit.diff.DiffEntry.ChangeType
 import org.eclipse.jgit.junit.RepositoryTestCase
-import org.eclipse.jgit.revwalk.RevCommit
 import org.junit.Before
-import org.junit.Rule
 import org.junit.Test
-import org.junit.rules.TemporaryFolder
 
 import static org.assertj.core.api.Assertions.assertThat
-import org.eclipse.jgit.diff.DiffEntry.ChangeType
-import org.junit.After
 
 class GitServiceTest extends RepositoryTestCase {
-	
-
-	// @Rule public var tempFolder = new TemporaryFolder
 
 	@Inject GitService gitService // class under test
-	
-	
 	var Git git // filled by setup
 
 	@Before
 	def void setupTest() {
 		super.setUp
 		git = new Git(db)
-
 		// setup class under test
 		Guice.createInjector.injectMembers(this)
-		gitService.initRepository(git.repository.directory.parentFile)
+		gitService.openRepository(git.repository.directory.parentFile)
 	}
-	
+
 	@Test
 	def void testDiffOfModification() {
 		// given
-		
 		// -- initial commit
```

Comment:
consider factoring out into methods instead of having to have comments to provide structure.

Reply:
for test readability this is still ok, imho